tests/lib/test_oauth_util.py

```diff
@@ -197,26 +200,37 @@ async def test_verify_oauth_client_wrong_client_scope(db, config, user):
 
 
 class TestVerifyOauthClientRoles:
-    async def test_token_does_not_have_roles(self, db, config, user):
-        """Test no error even if there are not roles on the token"""
+    async def test_token_does_not_have_roles(self, db, config):
+        """Test that roles aren't required to be on the token - scopes can still be assigned directly"""
+        client, _ = ClientDetail.create_client_and_secret(
+            db,
+            CONFIG.security.oauth_client_id_length_bytes,
+            CONFIG.security.oauth_client_secret_length_bytes,
+            scopes=[PRIVACY_REQUEST_REVIEW],
+            user_id=None,
```

Comment:
This is one of the tests that was passing because the user's client had more scopes than the user did.   This is why the test needs more refactoring just to prove that scopes can still be assigned directly.